CRM/Contact/Form/Search.php

```diff
@@ -909,4 +909,16 @@ public function getTitle() {
     return ts('Search');
   }
 
+  /**
+   * Load metadata for fields on the form.
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  protected function loadMetadata() {
+    // @todo - check what happens if the person does not have 'access civicontribute' - make sure they
+    // can't by pass acls by passing search criteria in the url.
+    $this->addSearchFieldMetadata(['Contribution' => CRM_Contribute_BAO_Query::getSearchFieldMetadata()]);
```

Comment:
@eileenmcnaughton are we missing ContributionSoft entity here?

Reply:
well yes & no - yes in that it probably should be there in the end - but no in the sense that no ContributionSoft fields have been converted to metadata as yet

Reply:
I see